test/e2e/storage/persistent_volumes-local.go

```diff
@@ -53,98 +60,309 @@ const (
 	// TODO: This may not be available/writable on all images.
 	hostBase      = "/tmp"
 	containerBase = "/myvol"
-	testFile      = "test-file"
-	testContents  = "testdata"
-	testSC        = "local-test-storagclass"
+	discoveryDir  = "disks" // 'hostBase + discoveryDir' is the path for volume discovery.
+	// Path to the first volume in the test containers
+	// created via createLocalPod or makeLocalPod
+	// leveraging pv_util.MakePod
+	volumeDir = "/mnt/volume1"
+	// testFile created in setupLocalVolume
+	testFile = "test-file"
+	// testFileContent written into testFile
+	testFileContent = "test-file-content"
+	testSC          = "local-test-storageclass"
+
+	// Following are constants used for provisioner e2e tests.
+	//
+	// testServiceAccount is the service account for bootstrapper
+	testServiceAccount = "local-storage-bootstrapper"
+	// testRoleBinding is the cluster-admin rolebinding for bootstrapper
+	testRoleBinding = "local-storage:bootstrapper"
+	// volumeConfigName is the configmap passed to bootstrapper and provisioner
+	volumeConfigName = "local-volume-config"
+	// bootstrapper and provisioner images used for e2e tests
+	bootstrapperImageName = "quay.io/external_storage/local-volume-provisioner-bootstrap:latest"
+	provisionerImageName  = "quay.io/external_storage/local-volume-provisioner:v1.0.0"
```

Comment:
I saw we are using released version of nfs provisioner

https://github.com/kubernetes/kubernetes/blob/7deb49ecd8de101d4402bd51128fd0752f0b1d34/test/e2e/storage/volume_provisioning.go#L833

It depends on what we want in kubernetes CI.  If all we want is to make sure changes in kubernetes won't break provisioner, then i would go for released version; if we want to CI on provisioner itself, then we can change it to `latest`.